src/util/cookieUtils.js

```diff
@@ -0,0 +1,8 @@
+import querystring from 'qs';
+
+const isProd = process.env.NODE_ENV === 'production';
+export const MEMBER_COOKIE = isProd ? 'MEETUP_MEMBER' : 'MEETUP_MEMBER_DEV';
+
+export const parseMemberCookie = state =>
```

Comment:
definitely. I was wondering about the parseInt + fallback, but I was a little cautious about assuming the return value should be an `int` in all cases... but it makes sense to treat that as the default, and it can be re-stringified if need be